### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by orydatadudes** [stylegan-human/StyleGAN-Human#13]
can you publish your training code
thanks

**Comment by stylegan-human** [comment]
Hi, 
We trained the model using the official [Stylegan2-ada-pytorch](https://github.com/NVlabs/stylegan2-ada-pytorch) repo, with a changed input resolution.

**Comment by redyellowRain** [comment]
is it possible to use your pkl (SG2-Ada-Pytorch) file to further train the model with a different dataset . if yes can you please help me how to do so?

**Comment by redyellowRain** [comment]
Hello and thank you for your help! How did you change the input resolution in the official stylegan2-ada-pytorch file? Do you mind sharing the code how it was done? thank you...

**Comment by eliohead** [comment]
I'm also trying to train (finetune). Would be great the training script to be made public.

**Comment by EnricoBeltramo** [comment]
Yes, a sample should be helpful. I'm not clear how to change the original repo in order to have rectangular with same results.

**Comment by redyellowRain** [comment]
please make the code public so we can train in Google Colab on our own-- would be great to see what can be done with this Stylegan creatively.. please share
